src/itemChoice/index.tsx

```diff
@@ -107,7 +107,7 @@ class ItemChoice extends PureComponent <ItemChoiceProps> {
     return (
       <Component
         role="option"
-        key={key}
+        key={id}
```

Comment:
🤔 why do we have a key here ? IMO you should just remove it

Reply:
ping @Bidjii

Reply:
Yes you could remove it, I talked about it with Loïck, I just have to make sure that Typescript doesn't throw us an error if we don't allow the props to be put here on the first place

Reply:
Ok I removed it and checked, it's all good.